requirements.txt

```diff
@@ -29,3 +29,7 @@ trio==0.24.0
 trio-websocket==0.11.1
 urllib3 @ file:///private/var/folders/k1/30mswbxs7r1g6zwn8y4fyt500000gp/T/abs_068obtb882/croot/urllib3_1698257558009/work
 wsproto==1.2.0
+
+requests~=2.31.0
```

Comment:
please use == as an exact package version. It will increase the stability of your side-packages and prevent bugs in the future. 

If you want to bump requirements, please test it and update manually

Reply:
changed